README.md

```diff
@@ -0,0 +1,48 @@
+# PhpToken Polyfill
+
+A polyfill to bring the [PHP 7.3 `array_key_first`](https://www.php.net/manual/en/function.array-key-first.php) and [PHP 7.3 `array_key_last`](https://www.php.net/manual/en/function.array-key-last.php) functions to PHP 5.3 and later.
+
+[![Latest Stable Version](https://poser.pugx.org/phpwatch/array_key_first_last-polyfill/v/stable)](https://packagist.org/packages/phpwatch/array_key_first_last-polyfill) [![License](https://poser.pugx.org/phpwatch/array_key_first_last-polyfill/license)](https://github.com/phpwatch/array_key_first_last-polyfill)  [![CI](https://github.com/phpwatch/array_key_first_last-polyfill/workflows/CI/badge.svg)](https://github.com/phpwatch/array_key_first_last-polyfill/actions)
+
+## Synopsis
+
+This library provides a compatible layer to bring `array_key_first` and `array_key_last` functions that comes with PHP 5.3. They provide the full functionality, but from user-land PHP code. Note that PHP 7.3's built-in these above functions are fast and memory-friendly because it is written in C language. However, if you have a library that requires these functions, you can use this library to transparently bring that functionality to any PHP version 5.3 or later.
```

Comment:
```suggestion
This library provides a compatible layer to bring `array_key_first` and `array_key_last` functions that come with PHP 7.3. They provide the full functionality, but from user-land PHP code. Note that PHP 7.3's built-in these above functions are fast and memory-friendly because it is written in C language. However, if you have a library that requires these functions, you can use this library to transparently bring that functionality to any PHP version 5.3 or later.
```